### PR TITLE
Added ability to set ListPicker item color and background color #204

### PR DESCRIPTION
**Pull request by kkashi01**
@halatmit @jisqyv @afmckinney 

Related to issue / enhancement #204 
Test Server http://ai-tester.appspot.com
Companion: http://www.3nportal.com/temp/MITAI2CompanionListPicker.apk

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/ListPickerActivity.java [hunk 1]
```diff
@@ -10,12 +10,8 @@
 
 import android.content.pm.ActivityInfo;
 import android.util.Log;
-import android.view.KeyEvent;
-import android.view.View;
-import android.view.WindowManager;
-import android.widget.AdapterView;
-import android.widget.ArrayAdapter;
-import android.widget.EditText;
+import android.view.*;
```
Comment (jisqyv) [inline appinventor/components/src/com/google/appinventor/components/runtime/ListPickerActivity.java]: @kkashi01 Please use explicit imports. (I suspect IntelliJ or Eclipse did this for you).

File: appinventor/components/src/com/google/appinventor/components/common/YaVersion.java [hunk 2]
```diff
@@ -618,8 +618,8 @@ private YaVersion() {
   // - Added ShowFilterBar property
   // For LISTPICKER_COMPONENT_VERSION 8:
   // - Added title property
-
-  public static final int LISTPICKER_COMPONENT_VERSION = 8;
+  // For LISTPICKER_COMPONENT_VERSION 9 - Added ItemTextColor, ItemBackgroundColor
+  public static final int LISTPICKER_COMPONENT_VERSION = 9;
```
Comment (jisqyv) [inline appinventor/components/src/com/google/appinventor/components/common/YaVersion.java]: @kkashi01 You also have to update YOUNG_ANDROID_VERSION. I suggest that you merge the current master branch before working more on this file...

**Comment by kkashi01** [comment]
@jisqyv updates done. However, the commit shows more changes to YaVersion, but its based on latest code updates in master

**Comment by jisqyv** [comment]
@kkashi01 Strange. I'm also seeing the conflict in YaVersion. I'm not sure what is going on, but I'll get to the bottom of it.

File: appinventor/components/src/com/google/appinventor/components/runtime/ListPicker.java [hunk 3]
```diff
@@ -61,6 +63,13 @@
                                 // if left blank, the App Name is used instead
   private boolean resumedFromListFlag  = false; //flag so onResume knows if the resume was triggered by closing the listpicker activity
 
+  private YailList deletedItems = new YailList();
```
Comment (jisqyv) [inline appinventor/components/src/com/google/appinventor/components/runtime/ListPicker.java]: @kkashi01 What's this deletedItems thing? I don't find a reference to it anywhere. I have removed it from the copy I am testing.

**Comment by jisqyv** [comment]
I removed whitespace and fixed some indentation. I also removed the deletedItems variable from ListPicker. It tests out good and will be merged.

LGTM

**Comment by jisqyv** [comment]
@kkashi01 Caught another problem. When you added the documentation for the new properties, you added it to the "Button" documentation instead of the "listPicker" documentation. I'll fix it.

**Comment by jisqyv** [comment]
MERGED

**Comment by kkashi01** [comment]
@jisqyv Thanks. The deletedItems was a leftover and removing was proper. Thanks for update on html documentation

**Comment by jisqyv** [comment]
@kkashi01 Note: I also had to add an upgrader to versioning.js (a new requirement since you worked on this change).

**Pull request closed without merging**